### DISCUSSION TITLE
The impact of setting cache_timeout at database level

### DISCUSSION THREAD

**Discussion by snehlata08** [apache/superset#14049] [Q&A / Help]
I want to know if i set the cache_timeout at database level as 10 seconds how  is that going to affect the overall dashboard performances. Also If no one is using the system or viewing those particular dashboards is the cache still going to be overloaded every 10 seconds.

**Comment by kamalkeshavani-aiinside** [comment]
In my understanding, the cache timeout priority works in below order.
Dashboard>Chart>Dataset>Database>Config.py

So when you have not set cache timeout in Dashboard,Chart and dataset, then cache timeout setting of database will be used.

The cache is saved and used if the dashboard is loaded by anyone before cache gets timed out. In case of no one loading that dashboard, nothing will happen. But If you want to keep the cache refreshed even when no one is loading the dashboard, you need to use cache warmup strategy in celery.
Read more here:
https://superset.apache.org/docs/installation/cache

**Reply by snehlata08** [reply]
Thank you I will check this.
